homeassistant/components/octoprint/__init__.py

```diff
@@ -255,19 +273,27 @@ def get(self, endpoint):
                     self.job_error_logged = True
                     self.job_available = False
             elif endpoint == "printer":
-                log_string = f"Endpoint: printer {log_string}"
-                if not self.printer_error_logged:
-                    _LOGGER.error(log_string)
-                    self.printer_error_logged = True
+                if (
+                    status_code == 409
```

Comment:
Why filter it out, it is only going to get logged once because it's guarded by `self.printer_error_logged` ?

Reply:
I'm filtering out the 409 http status on Printer Endpoint because Octoprint API returns 409 on any printer status that is not operational(error, offline, etc.). In previous version it caused logging of error msg, every single request, anytime the printer is offline(disconnected in terms of Octoprint). I though that we can make it silent and the printer sensor itself will be set to state `unknown` when the printer is not in operational state.